.gitlab-ci.yml

```diff
@@ -374,6 +380,9 @@ pkg:nix:deploy:channel:
     - git fetch --unshallow
     - git branch -v
     - git push git@github.com:coq/coq-on-cachix "${CI_COMMIT_SHA}":"refs/heads/${CI_COMMIT_REF_NAME}"
+  only:
+    variables:
+      - $FULL_CI == "true"
```

Comment:
This is redundant with the above `only`, remove or merge.

Reply:
indeed